docs/index.mdx

```diff
@@ -31,6 +31,14 @@ Each platform provides different features to make an app more accessible. We've
     description="Android has an accessibility layer that helps users to navigate their Android devices more easily. By integrating accessibility features, you can improve your app's usability, particularly for users with disabilities."
     image="/img/light/Android.svg"
     alt="Android logo"
+  />
+    <LinkListItem
+    url="/en/docs/compose"
+    title="Learn about Jetpack Compose accessibility"
+    label="Jetpack Compose accessibility"
+    description="Jetpack Compose has an accessibility layer that helps users to navigate their Android devices more easily. By integrating accessibility features, you can improve your app's usability, particularly for users with disabilities."
```

Comment:
Change to:

Jetpack Compose offers accessible building blocks to create inclusive interfaces. Key concepts like semantics, traversal and accessibility actions enhance usability for everyone.